recipes-devtools/ocaml/ocaml-cross_4.04.2.bb

```diff
@@ -42,7 +42,6 @@ do_configure_x86() {
                 -cc "${TARGET_PREFIX}gcc -fPIC -m32 --sysroot=${STAGING_DIR_TARGET}" \
                 -as "${TARGET_PREFIX}as --32" \
                 -aspp "${TARGET_PREFIX}gcc -fPIC -m32 -c" \
-                -libs "-Wl,--sysroot=${STAGING_DIR_TARGET}" \
```

Comment:
I don't really know this code, but -libs was added for a reason, why can we remove it now?  The commit message only mentions the sysroots which state the problem.  I think it would be helpful to have an explanation of why it can be removed and/or is no longer needed.  (Is it reverting from static to dynamic linking?)

Reply:
This one is a bit confusing, it is taken by the `configure` directly and appends `cclibs`, a variable then used to compose `BYTECCLIBS` and `NATIVECCLIBS`, both used to pass down linker arguments to `gcc` from the bytecode and native compilers.

Initially, without this, `ocaml-cross` compilers would not pass `--sysroot=${STAGING_DIR_...}`, resulting in native or even host libraries to be used (OXT-1704 and alike). Unfortunately, setting this in the `ocaml-cross` recipes was misleading as ` STAGING_DIR_TARGET` is expanded to that recipe's sysroot.

Compilation/linker options for the sysroot are now passed by the `ocaml.bbclass` through `OCAMLC`, `OCAMLOPT`, `OCAMLMKLIB`, which are defined for each recipe/class.

Reply:
"Compilation/linker options for the sysroot are now passed by the ocaml.bbclass through OCAMLC, OCAMLOPT, OCAMLMKLIB, which are defined for each recipe/class."
This would be sufficient for me. Thanks.